lib/rules/declaration-property-value-no-unknown/__tests__/index.mjs

```diff
@@ -62,6 +62,9 @@ testRule({
 		{
 			code: 'a { overflow: overlay; }',
 		},
+		{
+			code: 'a { overflow-y: overlay; }',
```

Comment:
[minor]
I  think it would be good to include a test for `overflow-x` as well since it overrides the syntax.

Reply:
```suggestion
			code: 'a { overflow-x: -moz-scrollbars-none; overflow-y: overlay; }',
```